pkgs/games/openarena/default.nix

```diff
@@ -31,6 +38,9 @@ stdenv.mkDerivation {
     makeWrapper "${gameDir}/openarena.${arch}" "$out/bin/openarena" \
       --prefix LD_LIBRARY_PATH : "${libPath}"
     makeWrapper "${gameDir}/oa_ded.${arch}" "$out/bin/oa_ded"
+
+    cd openarena-0.8.8
+    unzip $communityMaps
```

Comment:
TODO: Delete sources/ which is ~70% of the unzipped space.

Reply:
Done.

Reply:
I think this fits better into the unpackPhase.